lib/cocoapods-core/specification/dsl.rb

```diff
@@ -1261,15 +1261,16 @@ def dependency=(args)
 
       # @!method vendored_frameworks=(*frameworks)
       #
-      #   The paths of the framework bundles that come shipped with the Pod.
+      #   The paths of the framework bundles that come shipped with the Pod. Supports both `.framework` and `.xcframework` bundles.
+      #   The frameworks will be made available to the Pod and to the consumers of the pod.
```

Comment:
chose this language to avoid specifying exact implementation details that may change in the future